queries.sql

```diff
@@ -179,4 +179,11 @@ JOIN species ON animals.species_id = species.id
 WHERE vets.name = 'Maisy Smith'
 GROUP BY species.name
 ORDER BY COUNT(species.name) DESC
-FETCH FIRST 1 ROW ONLY;
\ No newline at end of file
+FETCH FIRST 1 ROW ONLY;
+
+
+-- Performance
+
+EXPLAIN ANALYSE SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYSE SELECT * FROM visits where vet_id = 2;
```

Comment:
- [OPTIONAL] Even if the execution times difference is very small for this query:

```SQL
EXPLAIN ANALYSE SELECT * FROM visits where vet_id = 2;
```
In your case, I can clearly see that you had some fun with the `explain analyze` SQL statement and I am sure that in the future you will know how to start the investigation for some slow DB queries 💻 🖱️